### PR TITLE
UI : Added Organization displayName in DataInsight health glance

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue : #8996
- Added Organization displayName in DataInsight health glance

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by ShaileshParmar11 — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx [hunk 1]
```diff
@@ -60,6 +62,10 @@ const DataInsightSummary: FC<Props> = ({ chartFilter, onScrollToChart }) => {
 
   const [mostActiveUser, setMostActiveUser] = useState<MostActiveUsers>();
 
+  const [OrganizationDetails, setOrganizationDetails] = useState<Team>(
+    {} as Team
+  );
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx:67]:
```suggestion
  const [OrganizationDetails, setOrganizationDetails] = useState<Team>(
  );
```

File: openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx [hunk 2]
```diff
@@ -72,6 +78,11 @@ const DataInsightSummary: FC<Props> = ({ chartFilter, onScrollToChart }) => {
 
   const { t } = useTranslation();
 
+  const fetchOrganizationDetails = async () => {
+    const data = await getTeamByName('Organization');
+    setOrganizationDetails(data);
+  };
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx:84]: Wrap with try/catch

File: openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx [hunk 3]
```diff
@@ -169,7 +181,10 @@ const DataInsightSummary: FC<Props> = ({ chartFilter, onScrollToChart }) => {
       loading={isLoading}
       title={
         <Typography.Title level={5}>
-          {t('label.data-insight-summary')}
+          {t('label.data-insight-summary', {
+            organization:
+              OrganizationDetails.displayName || t('label.openMetadata'),
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx:186]: @Ashish8689, We should use `getEntityName` util, so if displayName is not present it fallback to name.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 4]
```diff
@@ -552,7 +552,8 @@
     "select-dbt-source": "Select DBT Source",
     "no-selected-dbt": "No source selected for DBT Configuration.",
     "dbt": "DBT",
-    "latest": "Latest"
+    "latest": "Latest",
+    "openMetadata": "OpenMetadata"
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:556]: @Ashish8689 , We already have `the "open-metadata"` key, use that.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Comment by Ashish8689** [comment]
@Sachin-chaurasiya  changes made as per comments!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**